hebsafeharbor/common/prepositions.py

```diff
@@ -1,3 +1,4 @@
 LOCATION_PREPOSITIONS = ["מ", "ל", "ב", "ו"]
 DISEASE_PREPOSITIONS = ["מ", "ל", "ב", "ו"]
 MEDICATION_PREPOSITIONS = ["מ", "ל", "ב", "ו"]
+MEDICAL_TEST_PREPOSITIONS = ["מ", "ל", "ב", "ו"]
```

Comment:
Why don’t you just use HEBREW_PREPS?

Reply:
in case that it will be different between them (need to consider more specific lists after deeper evaluation)